pages/checkout/_subscription.html

```diff
@@ -0,0 +1,131 @@
+          </ul>
+        </div>
+      </div>
+      <div class="col-md-4 panel panel-default checkout-panel-right">
+        <div class="row">
+          <div class="col-xs-6 col-sm-12">
+            <h6 class="margin-top-none text-center visible-sm-block visible-md-block visible-lg-block">
+              <strong>Your Subscription</strong>
+            </h6>
+            <div class="checkout-price-view">
+              <div class="checkout-price text-center" data-checkout-total="default">
+                <small class="visible-xs-block" style="margin-bottom: -4px">Your Total Today</small>
+                <span class="large-price visible-xs-block" style="white-space: nowrap;"><span>$</span><strong id="subscription-subtotal"></strong></span>
+                <div class="visible-sm-block visible-md-block visible-lg-block">
+                  <span class="large-price"><span>$</span><strong id="subscription-price"></strong></span>&nbsp; / month
+                  <p class="text-center" id="checkout-price-addon" style="display: none;"><span class="large-price large-price-sub"><span>+$</span>4,950</span>&nbsp; flat fee</p>
+                </div>
+              </div>
+              <div class="checkout-price-enterprise" data-checkout-total="enterprise" style="display: none;">
+                <h5 class="h1 large-price large-price-sub text-center margin-top-none"><span>Enterprise</span></h5>
+              </div>
+            </div>
+            <hr class="hr-sm hr-reset visible-sm-block visible-md-block visible-lg-block">
+            <div class="visible-sm-block visible-sm-block visible-md-block visible-lg-block">
+              <h5 class="margin-top-none text-center">
+                <strong>What's Included:</strong>
+              </h5>
+              <div class="row">
+                <div class="col-xs-12 col-sm-6 col-sm-offset-3 col-md-12 col-md-offset-0">
+                  <ul class="check-list small text-white margin-bottom-none">
+                    <li><span class="help-text light" title="250,000+ lines of reusable, battle-tested infrastructure code." data-toggle="tooltip">Infrastructure as Code Library</span></li>
+                    <li><span class="help-text light" title="A collection of training video courses that teach you DevOps." data-toggle="tooltip">DevOps Training Library</span></li>
+                    <li><span class="help-text light" title="Mission Control for your infrastructure. Currently in private beta." data-toggle="tooltip">Gruntwork Houston</span></li>
+                    <li><span class="help-text light" title="Get support via our public community forum. 2 Business Day SLA." data-toggle="tooltip">Community Forum</span></li>
+                    <li class="check-list-disabled" id="subscription-addon-1">Dedicated Support</li>
+                    <li class="check-list-disabled" id="subscription-addon-2">Reference Architecture</li>
```

Comment:
Support and Ref Arch should have the dotted underline for consistency

Reply:
+1. @sahilakos Could you add this?

Reply:
Will do.